upgrade.sh

```diff
@@ -113,13 +113,33 @@ fi
 # Silently create default docker compose env file if needed.
 if [ ! -f ${COMPOSE_ENV_FILE} ]; then
     cp "${COMPOSE_ENV_FILE}.sample" "${COMPOSE_ENV_FILE}"
+else
+    # Convert GRAFANAUSER to PWD_USER in existing compose env file (required in 2.10.0)
+    sed -i "s@GRAFANAUSER@PWD_USER@g" "${COMPOSE_ENV_FILE}"
+    if grep -q "^PWD_USER=\"1000:1000\"" "${COMPOSE_ENV_FILE}"; then
+        sed -i "s@^PWD_USER=\"1000:1000\"@#PWD_USER=\"1000:1000\"@g" "${COMPOSE_ENV_FILE}"
+    fi
```

Comment:
Nice!  Here is what this does:

* It replaces all occurrences of "GRAFANAUSER" with "PWD_USER" in the `compose.env` file.
* It checks if a line starting with "PWD_USER=" followed by "1000:1000" exists in the file. If it does, it comments out that line by adding a "#" at the beginning. This is because the `powerwall.env` will default to that.

Reply:
Yep - all defaults are set in the `powerwall.yml` docker compose file now when a variable is undefined.